Source/WebCore/platform/network/cf/DNSResolveQueueCFNet.cpp

```diff
@@ -116,6 +118,11 @@ void DNSResolveQueueCFNet::performDNSLookup(const String& hostname, Ref<Completi
     RetainPtr hostEndpoint = adoptCF(nw_endpoint_create_host(hostname.utf8().data(), "0"));
     RetainPtr context = adoptCF(nw_context_create("WebKit DNS Lookup"));
     RetainPtr parameters = adoptCF(nw_parameters_create());
+
+    auto bundleID = applicationBundleIdentifier();
+    if (!bundleID.isEmpty())
```

Comment:
This could be one line:
if (auto bundleID = applicationBundleIdentifier(); !bundleID.isEmpty())
Does it matter?  Not really.